### DISCUSSION TITLE
Team up! (Find potential teammates here) 👩🏽‍🤝‍👨🏻

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/AI-Chat-App-Hack#6] [General]
We've heard from some participants that they'd like to find team members. This is a good opportunity to combine skills! If you're looking for a team member, please reply with:

* What skills you're looking for (if any)
* Your timezone/availability to meet
* Preferred form of communication (email, discord, etc)
* Some way to contact you

**Comment by pallavigadepalli** [comment]
Hello! I was looking for a FE and BE developer for a conversational AI tool. I did find someone but had some questions. 
1. I had registered earlier as an individual, how can I now re-register as a team?
2. I am guessing this is a remote event since my team is not geographically co located. 
3. In addition, this hackathon is from Jan 29 through Feb 12, Is it only because its live streaming or do you need to build it during that timeframe?
4. We are four in all and the page suggests three to a team, is that a strict rule?

**Reply by CharanBharathula** [reply]
Hi. I have skill of FE development

**Reply by Venkatakrishnan-Ramesh** [reply]
Interested as a AI-Developer

**Reply by vdekercd** [reply]
I can help you as BE developer :)

**Comment by NitkarshChourasia** [comment]
Skills: Must be Proficient in applied Python syntax.
Timezone: Let's Discuss and decide a common time zone. IST (Indian Standard Time - UTC+5:30).
Preferable Communication Mediums: Gmail, Telegram, WhatsApp.
Contact: {"email_address": "playnitkarsh@gmail.com", "telegram_username": "https://t.me/NitkarshChourasia"} - Reach out to discuss, and we can continue on your preferred platform from the HashMap or switch to WhatsApp if you prefer that as convenient medium.

Thank you
@NitkarshChourasia

**Reply by CharanBharathula** [reply]
I have good proficiency on python

**Reply by Venkatakrishnan-Ramesh** [reply]
I have good experience with python and AI

**Reply by shiv2-dot** [reply]
I have good exposure to python and AI.

**Comment by KomaliValluru** [comment]
Hello everyone, I'm looking for a teammate. email me on: vkomalidevi9866@gmail.com. I've got some ideas and looking to discuss them.

**Comment by pallavigadepalli** [comment]
Folks - can someone team up with me I am struggling to keep up and even fork the repo. brand new person to python,  non-developer. Can someone help me out I want to catch up to day 2 and really deploy code and start on this. Prolly a 15 minutes call on zoom will suffice. thanks!

**Reply by multispark** [reply]
I suggest taking a look at the first livestream: https://www.youtube.com/watch?v=hX1yaUBpM4s

Also will suggest taking a look at the free deployment video that Pamela made yesterday: https://www.youtube.com/watch?v=nlIyos0RXHw

**Reply by pallavigadepalli** [reply]
will try that and see. just very new to this so overwhelmed. 
and trying to team with someone else like me to figure this out @multispark

**Reply by pallavigadepalli** [reply]
@multispark , i have logged into portal.azure.com - what do I do first? what type of resource should I be creating? in the free deployment, pamela talks about getting a free account but I already have some credits so plan on using that to not have limitations. Can you help? Issue is I don't see a chat  playground in my view.

**Reply by zedhaque** [reply]
Is this the first time you are logged into portal.azure.com? Do you have owner or admin level permissions? Can you visit 

https://oai.azure.com/

If issues it means you need to setup OpenAI service as well as various GPT models to be able to user open AI studio - Here is a step by step tutorial. It's the no-code option but you will need higher level permission (account owner/admin) and access to Azure Open AI (separate application/approval process from Microsoft)

https://techcommunity.microsoft.com/t5/educator-developer-blog/bring-your-own-data-to-azure-openai-step-by-step-guide/ba-p/3905212

**Comment by pallavigadepalli** [comment]
Second time probably, Will try this step by step version.
Here is the error I am seeing looks like a subscription error and I cannot
figure out the reason why.

here is what I see on portal.azure.com 
![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/156367876/b012d933-e511-4d29-a46d-d34508ba77af)

and when I go to oai i get asked this: 

![Screenshot 2024-01-30 at 4 49 20 PM](https://github.com/microsoft/AI-Chat-App-Hack/assets/156367876/1f748584-36f7-4ff4-be44-e228f1961d8e)

Unsure on how to proceed and link you shared.

**Comment by pallavigadepalli** [comment]
I am the owner -pretty sure I have admin level permissions, how would I
know that?

**Comment by zedhaque** [comment]
Below URL showcases Azure subscriptions and you will find your role under "My Role" in the table.

https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBladeV2

https://portal.azure.com/#view/Microsoft_Azure_GTM/ModernBillingMenuBlade/~/BillingAccounts

As you are the owner you can follow the above blog URL also all the discussion below the article and you should be good to go. Again this is the no-code solution.

**Comment by pallavigadepalli** [comment]
Zed
I just double checked, I am the admin.

I am following the docs but while going through the steps of creating the
AI search service it tells me I will be charged 200 some dollars a month.
Is that usual?

[image: image.png]

I also have three subscriptions and wondering how to delete the ones that
don't matter. especially since I have credits from MS founders hub. that is
causing some issues and confusion.

Pallavi

**Reply by zedhaque** [reply]
Blog post asks for you to choose the "Standard" pricing tier and the charge you mention is about correct. But you have options

Select "Standard" only if you have 25GB worth of data to index/upload (note - your resource group name - cos at any time you can delete the resource group and all services will be deleted) - this option probably will cost you $10 for you to try out building the solution). 

In the "Standard" drop down you can also choose "Basic" if you want to upload 2GB worth of data. Monthly cost than drops to $80 or so. I would create a new resource group for this.

You can also create a "Free" tier - which gives you 50MB - you probably can choose this first to try out the blog post (but this tier has limitations so you might run into other issues) - regardless you can try this and get familiar with how to set things up and the blog post as a whole. Again you should create a new resource group so you can delete it to avoid any cost. 

Also just create a new resource group and then delete it so you know how to delete resource groups.

**Reply by pallavigadepalli** [reply]
that is correct I have created a brand new resource group called hackathon. at least for the hackathon I wouldnt want to be charged which is why the question. I did not get the free options at all. I will check for basic and delete it after hakcathon so will be charged minimally. thanks!

**Reply by pallavigadepalli** [reply]
Made a mistake i want to delete all resource groups till I get the subscriptions figured out.  " delete " icon on the group is disabled. Any idea how to delete all resource groups thus far. Looks like different resources created are in different resource groups.

**Reply by zedhaque** [reply]
You need to select a subscription (Azure subscription 1) where Azure Open AI access with SKU 'S0' was granted. This is a separate application/approval process. I think it takes about a day to get approval. Maybe you have other Subscriptions which has such approval.

**Reply by pallavigadepalli** [reply]
it is granted against azure subscription 1.

**Reply by pallavigadepalli** [reply]
@zedhaque  i think I am finally successful. I half created using low code and half did it with github codespaces. Success I think... check out the screenshot but need to know what to do next. 
![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/156367876/da4844cc-bf01-4c33-b64b-62a092d8606b)

**Reply by zedhaque** [reply]
Awesome :) persistence paid off :)

I guess you have clicked on the link and tried those example questions. Your screenshot should look like the Hack session YouTube and you should get answers/chat. You are using the sample docs from the repo. You can ask questions from the doc in the data folder.
Great work!

**Reply by pallavigadepalli** [reply]
Yes I have clicked on the domain website link to try to answer the sample questions. The front end looks like the landing page and something that goes into a conversation when you enter text into the chat box. CAn you recommend which doc in the data folder has that? 

I am moving on to the session 2 to see what it asks and session 3 was the meat of the it so will look at that today. 
QQ - chris Ayers said he will check in code to even read ppt docs. Can you let me know when he would be planning to do that?
